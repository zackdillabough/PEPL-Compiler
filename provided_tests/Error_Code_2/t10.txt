POLY A(M, N, O) = M^7 - 6 N M O^4 + 9 M^8 N^3 O^2 - 5 N M^5 O + 23 M^2 N O;
POLY 
B = 
y^4 
+ x^3 + y^2 +
 x^1 + x^0;
POLY C(P, Q) = P - P + 2 Q - Q + R^473;
POLY E = X + x^2;
POLY F(a, b, s, c, e, r, y) = a - b + c - d + e - f;
POLY U(X, Y, x) = x + X + Y;
START
INPUT X;
INPUT Y;
A(X, Y, X);
B(B(B(X)));
INPUT X;
INPUT Y;
C(X, Y);
98 34 92 12 3 4
