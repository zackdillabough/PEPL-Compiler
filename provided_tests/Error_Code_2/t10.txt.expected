Error Code 2: 4 5 7 8 9 9 